build-image/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:8
+
+RUN apt-get -y update
+
+RUN apt-get -y install maven
+
+
```

Comment:
this file probably isn't necessary, we have `docker.gameofloans.com/universe/maven:3.5-jdk-8`

Reply:
It's for reference for the future developers.  I spent a considerable amount of time trying to come up with a build environment that could do the build.